libp2p-tcp-transport/src/lib.rs

```diff
@@ -237,7 +237,7 @@ mod tests {
             let listener = tcp.listen_on(addr).unwrap().0.for_each(|(sock, _)| {
                 // Define what to do with the socket that just connected to us
                 // Which in this case is read 3 bytes
-                let handle_conn = tokio_io::io::read_exact(sock, [0; 3])
+                let handle_conn = tokio_io::io::read_exact(sock.unwrap(), [0; 3])
```

Comment:
Ideally this would be an `expect`, since if this fails we'll just turn it into an `expect` in order to find out what failed anyway.

Reply:
I didn't put an `expect` because we're in a test and `Err` is actually never produced at all. I can put one if needed, but that seems unnecessary.

Reply:
expect would be better, but given that tests are run in debug mode (w/ backtraces) I think it's fine.

Any non-test code should expect instead of unwrap, since debuginfo won't be available.